src/PKSim.Core/Model/Simulation.cs

```diff
@@ -19,6 +20,25 @@ public abstract class Simulation : PKSimBuildingBlock, ISimulation, IWithChartTe
       private readonly List<ISimulationAnalysis> _allSimulationAnalyses = new List<ISimulationAnalysis>();
       private SimulationProperties _properties;
       private SimulationResults _results;
+      private readonly ICache<string, CompoundPK> _compoundPKCache;
+
+      public virtual DataColumn PeripheralVenousBloodColumn(string compoundName)
```

Comment:
I think this should be abstract. But I am actually not sure that this belongs here at all. Why are we implementing something specific for just a few outputs? Does not make much sense to me

Reply:
It was already defined at IndividualSimulation and although you are right, it makes some sense since we are using this specific combination ([peripheral] blood plasma) a lot. It could be on a service though but keep in mind that it is extracted differently at Population/Individual Simulation. Also, it was not abstract since there are more implementations of Simulation and there it is not required but again, makes sense that it is abstract here and only on the concrete implementations we decide whether it returns something or not.